debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/generated/PlSqlParser.g4

```diff
@@ -3351,7 +3351,7 @@ foreign_key_clause
     ;
 
 references_clause
-    : REFERENCES tableview_name paren_column_list
+    : REFERENCES tableview_name paren_column_list? (ON DELETE (CASCADE| SET NULL_))?
```

Comment:
@Naros Is the underscore in the `NULL` intentional?
Could you please add an example that would verffy the added grammar fragment?
Could you pleas eprovide an upstream PR?
Thanks

Reply:
Hi @jpechane, yes the underscore is intentional, the Oracle grammar specifically uses `NULL_` as the lexical token for `NULL`.